music21/scale/__init__.py

```diff
@@ -289,6 +289,15 @@ def buildNetworkFromPitches(self, pitchList):
         >>> abstractScale._net.realizePitch('D5')
         [<music21.pitch.Pitch D5>, <music21.pitch.Pitch F#5>,
          <music21.pitch.Pitch A#5>, <music21.pitch.Pitch D6>]
+
+        Also possible with implicit octaves:
+
+        >>> abstract_scale = scale.AbstractScale()
+        >>> abstract_scale.buildNetworkFromPitches(['C', 'F'])
+        >>> abstract_scale.octaveDuplicating
+        True
+        >>> abstract_scale._net.realizePitch('G')
```

Comment:
btw -- moving _net to a public property is something I've wanted to do.  OR moving its contents directly into Scale.  Nobody knows whats in it.  Even me.